front/package.json

```diff
@@ -81,6 +81,7 @@
     "lodash": "^4.17.21",
     "lucide-react": "^0.363.0",
     "luxon": "^3.4.4",
+    "lru-memoizer": "^2.2.0",
```

Comment:
Don't we have pre-existing mechanism to achieve exactly that?

Reply:
- We had something that cache in redis, i just wanted to cache +/- during the execution of the request (which was way more complicated to do, so 3 seconds in local memory is good enough)
- We already had the lru-memoizer (as part of a jwt's dep) so nothing is really added here.

Reply:
I know, the lodash memoize method has no concept of ttl so it felt a bit dangerous to me as I'm not sure what's the lifetime of the lodash cache. The memoization cache from lodash is potentially shared between requests.
I don't want cached values to persist too long.